docs/contributing/development_environment.md

```diff
@@ -378,11 +378,13 @@ sudo openstack tripleo deploy \
   --standalone-role Standalone \
   -e /usr/share/openstack-tripleo-heat-templates/environments/standalone/standalone-tripleo.yaml \
   -e /usr/share/openstack-tripleo-heat-templates/environments/low-memory-usage.yaml \
+  -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm.yaml \
```

Comment:
Should we mix that change with this PR or submit an independent one?

Reply:
Whops, feel free to disregard. I thought https://github.com/openstack-k8s-operators/data-plane-adoption/pull/59 had merged already. If it merges first it can be addressed via rebase. If it merges after it will address it.